src/Microsoft.OData.Core/Microsoft.OData.Core.cs

```diff
@@ -7,8 +7,7 @@
 // </copyright>

```

Comment:
but the style like original 
namespace Microsoft.OData.Core
{

changed to 		 

namespace Microsoft.OData.Core {